packages/network-graph/components/network-graph/sampleData.ts

```diff
@@ -0,0 +1,370 @@
+    { source: 'Babet', target: 'Thenardier', value: 6 },
+    { source: 'Babet', target: 'Gueulemer', value: 6 },
+    { source: 'Babet', target: 'Valjean', value: 1 },
+    { source: 'Babet', target: 'Mme.Thenardier', value: 1 },
+    { source: 'Babet', target: 'Javert', value: 2 },
+    { source: 'Babet', target: 'Gavroche', value: 1 },
+    { source: 'Babet', target: 'Eponine', value: 1 },
+    { source: 'Claquesous', target: 'Thenardier', value: 4 },
+    { source: 'Claquesous', target: 'Babet', value: 4 },
+    { source: 'Claquesous', target: 'Gueulemer', value: 4 },
+    { source: 'Claquesous', target: 'Valjean', value: 1 },
+    { source: 'Claquesous', target: 'Mme.Thenardier', value: 1 },
+    { source: 'Claquesous', target: 'Javert', value: 1 },
+    { source: 'Claquesous', target: 'Eponine', value: 1 },
+    { source: 'Claquesous', target: 'Enjolras', value: 1 },
+    { source: 'Montparnasse', target: 'Javert', value: 1 },
+    { source: 'Montparnasse', target: 'Babet', value: 2 },
+    { source: 'Montparnasse', target: 'Gueulemer', value: 2 },
+    { source: 'Montparnasse', target: 'Claquesous', value: 2 },
+    { source: 'Montparnasse', target: 'Valjean', value: 1 },
+    { source: 'Montparnasse', target: 'Gavroche', value: 1 },
+    { source: 'Montparnasse', target: 'Eponine', value: 1 },
+    { source: 'Montparnasse', target: 'Thenardier', value: 1 },
+    { source: 'Toussaint', target: 'Cosette', value: 2 },
+    { source: 'Toussaint', target: 'Javert', value: 1 },
+    { source: 'Toussaint', target: 'Valjean', value: 1 },
+    { source: 'Child1', target: 'Gavroche', value: 2 },
+    { source: 'Child2', target: 'Gavroche', value: 2 },
+    { source: 'Child2', target: 'Child1', value: 3 },
+    { source: 'Brujon', target: 'Babet', value: 3 },
+    { source: 'Brujon', target: 'Gueulemer', value: 3 },
+    { source: 'Brujon', target: 'Thenardier', value: 3 },
+    { source: 'Brujon', target: 'Gavroche', value: 1 },
+    { source: 'Brujon', target: 'Eponine', value: 1 },
+    { source: 'Brujon', target: 'Claquesous', value: 1 },
+    { source: 'Brujon', target: 'Montparnasse', value: 1 },
+    { source: 'Mme.Hucheloup', target: 'Bossuet', value: 1 },
+    { source: 'Mme.Hucheloup', target: 'Joly', value: 1 },
+    { source: 'Mme.Hucheloup', target: 'Grantaire', value: 1 },
+    { source: 'Mme.Hucheloup', target: 'Bahorel', value: 1 },
+    { source: 'Mme.Hucheloup', target: 'Courfeyrac', value: 1 },
+    { source: 'Mme.Hucheloup', target: 'Gavroche', value: 1 },
+    { source: 'Mme.Hucheloup', target: 'Enjolras', value: 1 },
+  ],
+};
+
+export const triangleData = {
+  nodes: [
+    { id: 'node1', label: 'Node 1' },
+    { id: 'node2', label: 'Node 2' },
+    { id: 'node3', label: 'Node 3' },
+  ],
+  links: [
+    { source: 'node1', target: 'node2' },
+    { source: 'node2', target: 'node3' },
+    { source: 'node3', target: 'node1' },
+  ],
+};
+
+// export function coloredNodes() {
```

Comment:
can we clean up the commented out code here? or is it needed?

Reply:
Done.